lib/generators/hyrax/work/templates/locale.es.yml.erb

```diff
@@ -0,0 +1,37 @@
+en:
+  hyrax:
+    icons:
+      <%= file_name %>:     'fa fa-file-text-o'
+    select_type:
+      <%= file_name %>:
+        # TODO: translate `human_name` into Spanish
+        name:               "<%= human_name.titleize %>"
+        # TODO: translate `human_name` into Spanish
+        description:        "<%= human_name %> trabajos"
+  simple_form:
+    labels:
+      <%= file_name %>:
```

Comment:
The drawback is that we potentially have multiple places to fix/add translations. The benefit is that downstream users already have the translations in their apps, which makes it easier to tweak.